packages/kit/src/exports/vite/index.js

```diff
@@ -501,6 +517,134 @@ async function kit({ svelte_config }) {
 						}
 					`;
 				}
+
+				// The virtual module that is imported in the environment entrypoint files. This provides all the data that is needed to create the `Server` instance.
+				// Not implemented:
+				// - Server assets. The `read` function from `$app/server` can only be used in environments that support file system access.
+				// - Inlining styles. This requires communicating with the main process to collect dependencies. It should be possible (e.g. using import.meta.hot) but needs more investigation.
+				case sveltekit_environment_context: {
+					const { manifest_data, env, remote_address } = environment_context;
+
+					return dedent`
+						import { create_resolve } from "${runtime_base}/server/environment_context.js";
+
+						const resolve = create_resolve(${s(cwd)});
+
+						export let manifest = {
```

Comment:
should the manifest code be deleted from `vite/dev/index.js`? i.e. does this replace that one?

Reply:
Yes, same as above. This is the best approach but means that there can't be any holes in the implementation. It's quite useful to be able to compare with the existing code path for the time being but I'll remove the redundant code when possible.